devel/ci/integration/greenwave/start.sh

```diff
@@ -1,6 +1,7 @@
 set -m
 
-fedora-messaging consume &
+# the command on the next line is not present, possibly a version mismatch?
+# fedora-messaging consume &
```

Comment:
hum that seems wrong, we should try to understand why it is missing

Reply:
I can try to pin the version of the image.

Reply:
The image *is* pinned and after I removed it, I noticed integration tests still work. Reverted and will investigate in another issue.